docs/aws/audit/elbmonitoring/rules/elb_web_tier_health_check_remediation.mdx

```diff
@@ -1,6 +1,201 @@
+</Accordion>
+
+<Accordion title='Using CLI'>
+To ensure that the right health check configurations are used for web-tier Elastic Load Balancers (ELBs) in AWS using the AWS CLI, follow these steps:
+
+1. **Describe the Load Balancer:**
+   First, identify the ELB for which you want to configure the health check. Use the following command to list all your load balancers and get the necessary details:
+   ```sh
+   aws elb describe-load-balancers
+   ```
+
+2. **Configure Health Check:**
+   Use the `configure-health-check` command to set the appropriate health check parameters for your web-tier ELB. Ensure you specify the correct target, interval, timeout, unhealthy threshold, and healthy threshold. For example:
+   ```sh
+   aws elb configure-health-check --load-balancer-name <your-load-balancer-name> --health-check Target=HTTP:80/index.html,Interval=30,Timeout=5,UnhealthyThreshold=2,HealthyThreshold=2
+   ```
+
+3. **Verify Health Check Configuration:**
+   After configuring the health check, verify the settings to ensure they are correctly applied. Use the following command to describe the health check configuration:
+   ```sh
+   aws elb describe-load-balancers --load-balancer-names <your-load-balancer-name>
+   ```
+
+4. **Monitor Health Check Status:**
+   Regularly monitor the health check status of your instances to ensure they are healthy and the configurations are working as expected. Use the following command to describe the instance health:
+   ```sh
+   aws elb describe-instance-health --load-balancer-name <your-load-balancer-name>
+   ```
+
+By following these steps, you can ensure that the right health check configurations are used for your web-tier ELBs in AWS using the AWS CLI.
+</Accordion>
+
+<Accordion title='Using Python'>
+To prevent misconfigurations related to health check configurations for web-tier Elastic Load Balancers (ELBs) in AWS using Python scripts, you can use the Boto3 library, which is the AWS SDK for Python. Here are the steps to ensure the right health check configurations are used:
+
+1. **Install Boto3 Library**:
+   Ensure you have the Boto3 library installed. If not, you can install it using pip.
+   ```bash
+   pip install boto3
+   ```
+
+2. **Set Up AWS Credentials**:
+   Make sure your AWS credentials are configured. You can set them up using the AWS CLI or by creating a credentials file at `~/.aws/credentials`.
+
+3. **Create a Python Script to Configure Health Checks**:
+   Write a Python script to configure the health checks for your ELBs. Below is an example script that sets the health check configuration for an ELB.
+
+   ```python
+   import boto3
+
+   # Initialize a session using Amazon EC2
+   session = boto3.Session(
+       aws_access_key_id='YOUR_ACCESS_KEY',
+       aws_secret_access_key='YOUR_SECRET_KEY',
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code?t=secrets" target='_blank'>Click here for more details</a></b>